modules/led.sh

```diff
@@ -160,6 +160,8 @@ function led {
       ;;
     random)
       checkroot
+      echo "select from: "
```

Comment:
A little pedantic, but we could change "select from" to "selecting from" to be more precise.